swagger.json

```diff
@@ -1,325 +0,0 @@
-{
```

Comment:
Deleted this file as I copied over the changes to `.json` file in /tooling/ folder @srilavanyapaleti